packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -453,6 +455,7 @@ Future<T> showModalBottomSheet<T>({
   Clip clipBehavior,
   bool isScrollControlled = false,
   bool useRootNavigator = false,
+  bool isDismissible = true,
 }) {
   assert(context != null);
   assert(builder != null);
```

Comment:
Add an assert for `isDismissible != null` here

Reply:
Done.